### PR TITLE
Make Backspace key work as Delete key

### PR DESCRIPTION
**Pull request by moyogo**
See #7

### CODE REVIEW

File: Lib/defconQt/glyphCollectionView.py [hunk 1]
```diff
@@ -228,9 +228,9 @@ def keyPressEvent(self, event):
             self.selection = set()
         # XXX: this is specific to fontView so should be done thru subclassing of a base widget,
         # as is done in groupsView
-        elif key == Qt.Key_Delete:
+        elif key in (Qt.Key_Delete, Qt.Key_Backspace):
             #if self.characterDeletionCallback is not None:
-            if proceedWithDeletion() and self.selection:
+            if proceedWithDeletion(self) and self.selection:
```
Comment (adrientetar) [inline Lib/defconQt/glyphCollectionView.py]: Remove self in the function definition instead.

**Comment by adrientetar** [comment]
It should dispatch according to the platform, Backspace isn't usually used on Windows for this kind of operation.

**Comment by adrientetar** [comment]
I'll merge this manually.

**Comment by moyogo** [comment]
There are probably other behaviours that need be different according to the platform. How do you want to handle these?

**Comment by adrientetar** [comment]
Okay, I think I have sorted this out in master. Also, your earlier commit was the good one so I recovered it.

**Comment by moyogo** [comment]
Thanks!

**Pull request closed without merging**